src/client/app/utils/api/UploadCSVApi.ts

```diff
@@ -4,53 +4,75 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import ApiBackend from './ApiBackend';
+import { Dispatch } from '@reduxjs/toolkit';
+import { baseApi } from '../../redux/api/baseApi';
 import {
-	ReadingsCSVUploadPreferencesItem, MetersCSVUploadPreferencesItem, BooleanTypes, CSVUploadPreferencesForm, ReadingsCSVUploadPreferencesForm
+	BooleanTypes,
+	CSVUploadPreferencesForm,
+	MetersCSVUploadPreferencesItem,
+	ReadingsCSVUploadPreferencesForm,
+	ReadingsCSVUploadPreferencesItem
 } from '../../types/csvUploadForm';
+import ApiBackend from './ApiBackend';
+
+interface apiResponse {
```

Comment:
I'm curious as to why this was changed from starting capitalized. A search found lots of interfaces in OED and I did not easily see one this way. What do you think?